planning_scene_monitor_options:
  name: "planning_scene_monitor"
  robot_description: "robot_description"
  joint_state_topic: "/joint_states"
  attached_collision_object_topic: "/moveit_cpp/planning_scene_monitor"
  publish_planning_scene_topic: "/moveit_cpp/publish_planning_scene"
  monitored_planning_scene_topic: "/moveit_cpp/monitored_planning_scene"
  wait_for_initial_state_timeout: 10.0

planning_pipelines:
  pipeline_names: ["ompl", "pilz_industrial_motion_planner", "chomp", "ompl_rrt_star"]

plan_request_params:
  planning_attempts: 3
  planning_pipeline: ompl
  max_velocity_scaling_factor: 0.8
  max_acceleration_scaling_factor: 0.8

ompl_rrtc:
  plan_request_params:
    planning_attempts: 5
    planning_pipeline: ompl
    planner_id: "RRTConnectkConfigDefault"
    max_velocity_scaling_factor: 0.8
    max_acceleration_scaling_factor: 0.8
    planning_time: 2.0

ompl_rrt_star:
  plan_request_params:
    planning_attempts: 3
    planning_pipeline: ompl_rrt_star
    planner_id: "RRTstarkConfigDefault"
    max_velocity_scaling_factor: 0.8
    max_acceleration_scaling_factor: 0.8
    planning_time: 3.0

pilz_lin:
  plan_request_params:
    planning_attempts: 3
    planning_pipeline: pilz_industrial_motion_planner
    planner_id: "PTP"
    max_velocity_scaling_factor: 0.6
    max_acceleration_scaling_factor: 0.6
    planning_time: 1.5

chomp:
  plan_request_params:
    planning_attempts: 3
    planning_pipeline: chomp
    max_velocity_scaling_factor: 0.8
    max_acceleration_scaling_factor: 0.8
    planning_time: 2.5




